nat-lab/tests/test_batching.py

```diff
@@ -240,3 +242,77 @@ async def start_node_manually(client, node, sleep_s):
 
             print("Delay chart below")
             print(delay_chart)
+
+
+def proxying_peer_parameters(clients: List[ConnectionTag]):
+    def features():
+        features = default_features(enable_direct=False, enable_nurse=False)
+        features.wireguard.persistent_keepalive.proxying = 5
+        features.link_detection = FeatureLinkDetection(
+            rtt_seconds=2, no_of_pings=0, use_for_downgrade=False
+        )
+
+        features.batching = FeatureBatching(
+            direct_connection_threshold=5,
+            trigger_cooldown_duration=60,
+            trigger_effective_duration=10,
+        )
+        return features
+
+    return [
+        SetupParameters(
+            connection_tag=conn_tag,
+            adapter_type_override=TelioAdapterType.NEP_TUN,
+            features=features(),
+            fingerprint=f"{conn_tag}",
+        )
+        for conn_tag in clients
+    ]
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "setup_params",
+    [
+        proxying_peer_parameters(
+            [ConnectionTag.DOCKER_CONE_CLIENT_1, ConnectionTag.DOCKER_CONE_CLIENT_2]
+        )
+    ],
+)
+@pytest.mark.timeout(60)
+async def test_proxying_peers_reconnect(setup_params: List[SetupParameters]) -> None:
```

Comment:
There is no reconnection in this test ;)

Reply:
Whoops, that was initial experiment. Fixed that :)